swaap/swaap/View Controllers/ContactsViewController.swift

```diff
@@ -100,7 +138,11 @@ class ContactsViewController: UIViewController, ProfileAccessor, ContactsAccesso
 
 extension ContactsViewController: UITableViewDelegate, UITableViewDataSource {
 	func numberOfSections(in tableView: UITableView) -> Int {
-		fetchedResultsController.sections?.count ?? 0
+		fetchedResultsController.sections?.count ?? 1
```

Comment:
why does this default to 1?

Reply:
I was trying some different things trying to get collation to work

Reply:
Okay. Should probably get set back to 0 then. I believe this will crash if there are no entries.

Reply:
I will. But I think I might have figured out why we aren't getting any indexes. We had the sectionNamePath set to nil